docs/data/core-blocks.md

```diff
@@ -0,0 +1,86 @@
+# **core/blocks**: Block Types Data
+
+## Selectors 
+
+### getBlockType
+
+Returns a block type by name.
```

Comment:
"type by name" is a bit awkward, especially because "name" isn't really a type that I'd expect to provide clarification.

This might be better stated as "Return a block type as a string (eg. by name)." An example would be even better.